main/mafia.hs

```diff
@@ -144,7 +145,12 @@ run = \case
     mafiaScript path args
 
 parser :: Parser MafiaCommand
-parser = subparser . mconcat $ commands
+parser =
+  subparser (mconcat commands) <|> pScript
+
+pScript :: Parser MafiaCommand
+pScript =
+  MafiaScript <$> pScriptPath <*> many pScriptArgs
```

Comment:
confused - is this ever triggered? seems like this case is caught earlier by line 60